content/en/docs/distributions/minikf/minikf-gcp.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "Deploy Kubeflow using MiniKF on Google Cloud"
+title = "MiniKF on GCP Marketplace"
 description = "Deploy Kubeflow with MiniKF (mini Kubeflow) via Google Cloud Marketplace"
```

Comment:
I would remove the parenthesis altogether here:

"Deploy Kubeflow with MiniFK via GCP Marketplace"

so it is unified with the above.